packages/react-components/src/molecules/LabeledMultiSelect.tsx

```diff
@@ -30,9 +30,9 @@ const LabeledMultiSelect = <T extends MultiSelectOptionsType>({
   description,
   ...multiSelectProps
 }: LabeledMultiSelectProps<T>): ReactElement => (
-  <div css={{ paddingBottom: `${18 / perRem}em` }}>
+  <div css={{ paddingBottom: `${180 / perRem}em` }}>
```

Comment:
i didnt add 244px because it would create an unnecessary scrollable view(see below)

https://github.com/yldio/asap-hub/assets/37268411/22b45e93-8cc6-48ad-8e7c-e608a1e18bd6